multithreading.c

```diff
@@ -44,9 +45,9 @@ void *thr_func(void *arg) {
 
     //the enhanced random formula
     x=x1*RAND_MAX_P1+x2;
-    x=x/randmax_big;
+    x=x * randmax_big;
```

Comment:
just a small oversight here: should be `irandmax_big` instead of `randmax_big`. Also on line 50